index.js

```diff
@@ -46,7 +46,7 @@ function packAndDeploy(s3, bucket, path, force, callback){
     if(argv.dev) {
         if (fs.existsSync(path+'/.git')) {
             describe = exec("git describe --tags", {cwd: path}).stdout.replace('\n', '');
-            describe = describe || 'v0.0.0-' + exec('git rev-parse head', {cwd: path}).stdout.replace('\n', '');
+            describe = describe || 'v' + npmPackage.version + '-' + exec('git rev-parse head', {cwd: path}).stdout.replace('\n', '');
```

Comment:
while we're here can you change this to `git rev-parse HEAD`? This is an issue on linux machines https://github.com/mapbox/npm-internal/issues/15